src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,60 +4,115 @@
+
+    public RobotContainer() {
+        autoChooser = AutoBuilder.buildAutoChooser("Tests");
+        SmartDashboard.putData("Auto Mode", autoChooser);
+
+        configureBindings();
+    }
+
+    private void configureBindings() {
+        // Note that X is defined as forward according to WPILib convention,
+        // and Y is defined as to the left according to WPILib convention.
+        drivetrain.setDefaultCommand(
+                // Drivetrain will execute this command periodically
+                drivetrain.applyRequest(
+                        () ->
+                                drive.withVelocityX(
+                                                -joystick.getLeftY()
+                                                        * MaxSpeed) // Drive forward with negative Y
+                                        // (forward)
+                                        .withVelocityY(
+                                                -joystick.getLeftX()
+                                                        * MaxSpeed) // Drive left with negative X
+                                        // (left)
+                                        .withRotationalRate(
+                                                -joystick.getRightX()
+                                                        * MaxAngularRate) // Drive counterclockwise
+                        // with negative X (left)
+                        ));
+
+        joystick.a().whileTrue(drivetrain.applyRequest(() -> brake));
+        joystick.b()
+                .whileTrue(
+                        drivetrain.applyRequest(
+                                () ->
+                                        point.withModuleDirection(
+                                                new Rotation2d(
+                                                        -joystick.getLeftY(),
+                                                        -joystick.getLeftX()))));
+
+        joystick.pov(0)
+                .whileTrue(
+                        drivetrain.applyRequest(
+                                () -> forwardStraight.withVelocityX(0.5).withVelocityY(0)));
+        joystick.pov(180)
+                .whileTrue(
+                        drivetrain.applyRequest(
+                                () -> forwardStraight.withVelocityX(-0.5).withVelocityY(0)));
+
+        // Run SysId routines when holding back/start and X/Y.
+        // Note that each routine should be run exactly once in a single log.
+        joystick.back().and(joystick.y()).whileTrue(drivetrain.sysIdDynamic(Direction.kForward));
+        joystick.back().and(joystick.x()).whileTrue(drivetrain.sysIdDynamic(Direction.kReverse));
+        joystick.start()
+                .and(joystick.y())
+                .whileTrue(drivetrain.sysIdQuasistatic(Direction.kForward));
+        joystick.start()
+                .and(joystick.x())
+                .whileTrue(drivetrain.sysIdQuasistatic(Direction.kReverse));
```

Comment:
SysID stuff should be in test mode rather than in all modes (we don't want to do sysID during a match on accident)